### DISCUSSION TITLE
Cannot disable "successful" Action notifications

### DISCUSSION THREAD

**Discussion by owenconti** [community/community#28367] [Actions] [answered]
Starting earlier today, I've been receiving web notifications for successful action jobs. These are super annoying as I utilize my notifications for items requiring attention. 

It looks like there is a setting to revert back to the previous functionality which only notified for failed workflows. However, the checkbox is disabled unless I turn on email notifications. I should be able to disable successful notifications and still only receive web notifications.

<img width="526" alt="Xnapper-2022-08-04-10 43 47" src="https://user-images.githubusercontent.com/791222/182906153-94eb56d5-32e9-4a66-a8b4-5c34872b2b90.png">

**Comment by jfullstackdev** [comment]
why it's disabled ? 

not same as mine 

![image](https://user-images.githubusercontent.com/47092464/182911303-5b30f27d-a0bd-4375-8b4e-27c638d280aa.png)

if I unclick Web, then

![image](https://user-images.githubusercontent.com/47092464/182911633-c98dbfb5-1850-4ad7-b664-22048bd033f1.png)

**Comment by frenck** [comment, marked as answer]
I had the same issue and was absolutely flooded with notifications the last hours, super annoying.

I couldn't check the "Send notifications for failed workflows only" either (I never touched that setting as far as I am aware). However, Unchecking "Web" and next checking "Web" again, made it available.

After that I check to disable notifications for workflows and it seems to be silent (for now) 🙏 

../Frenck

**Reply by owenconti** [reply]
@frenck ~~thank you, this is what solved it for me! I didn't even think to try unchecking `Web` first 🤦‍♂️ (still seems like a bug though)~~

**Reply by owenconti** [reply]
Nevermind, ignore what I said. Unchecking and checking enables the "Send notifications for failed workflows only" checkbox but it does not save the selection:

![2022-08-04 11 52 43](https://user-images.githubusercontent.com/791222/182917871-68bd134e-03e1-4bd3-8c6f-df9bcc85f614.gif)

**Reply by frenck** [reply]
I see the same issue indeed, however, it does seem to save on my end? Maybe just a display issue?

![CleanShot 2022-08-04 at 19 57 21](https://user-images.githubusercontent.com/195327/182918696-a12090d5-ee27-4d19-ade0-ed8bcb6a9be3.png)

Its been pretty "silent" in my inbox (well the usual stuff, not the successful runs 😄 )

**Reply by owenconti** [reply]
@frenck just tested and same for me! So it does fix it but it isn't checking the box correctly when disabled.

Thanks for your help!

**Reply by ghost** [reply]
Also confirming that the box doesn't seem to represent reality and the "fix" is working.

**Comment by kingthorin** [comment]
I can confirm I also [started seeing this issue today](https://github.com/community/community/discussions/28377). I also experienced this weird behavior in the settings web UI. Will see if it resolves things.

**Comment by corneliusroemer** [comment]
I can confirm being flooded with notifications - I had disabled these in the past.

It seems to have stopped 8 hours ago though, so presumably Github realised the bug and fixed it.
